contracts/src/libraries/Constants.sol

```diff
@@ -11,3 +11,6 @@ uint64 constant NO_CHAL_INDEX = 0;
 
 // Expected seconds per block in Ethereum PoS
 uint256 constant ETH_POS_BLOCK_TIME = 12;
+
+// Legacy unsafe createRetryableTicket caller before call value check is implemented
```

Comment:
I dont fully understand this comment. Maybe you can make it a bit more long form

Reply:
```suggestion
// This address call Inbox.createRetryableTicket expecting unsafe behavior
// becuase it is deployed before the Inbox callvalue check is implemented
```

Reply:
`because` 😁

Reply:
edited